frontend/src/__tests__/cypress/cypress/e2e/acceleratorProfiles/AcceleratorProfiles.cy.ts

```diff
@@ -0,0 +1,83 @@
+import { mockDashboardConfig } from '~/__mocks__/mockDashboardConfig';
+import { mockDscStatus } from '~/__mocks__/mockDscStatus';
+import { mockStatus } from '~/__mocks__/mockStatus';
+import { acceleratorProfile } from '~/__tests__/cypress/cypress/pages/acceleratorProfile';
+import { mockK8sResourceList } from '~/__mocks__/mockK8sResourceList';
+import { mockAcceleratorProfile } from '~/__mocks__/mockAcceleratorProfile';
+import { deleteModal } from '~/__tests__/cypress/cypress/pages/components/DeleteModal';
+
+type HandlersProps = {
+  isEmpty?: boolean;
+};
+
+const initIntercepts = ({ isEmpty = false }: HandlersProps) => {
+  cy.intercept('/api/dsc/status', mockDscStatus({}));
+  cy.intercept('/api/status', mockStatus());
+  cy.intercept('/api/config', mockDashboardConfig({}));
+  cy.intercept(
+    '/api/k8s/apis/dashboard.opendatahub.io/v1/namespaces/opendatahub/acceleratorprofiles',
+    mockK8sResourceList(
+      isEmpty
+        ? []
+        : [
+            mockAcceleratorProfile({ displayName: 'Test Accelerator' }),
+            mockAcceleratorProfile({
+              name: 'some-other-gpu',
+              displayName: 'TensorRT',
+              enabled: false,
+              identifier: 'tensor.com/gpu',
+              description: 'Lorem, ipsum dolor sit amet consectetur adipisicing elit. Saepe, quis',
+            }),
+          ],
+    ),
+  );
+};
+
+describe('Accelerator Profile', () => {
+  it('empty state no accelerator profile', () => {
+    initIntercepts({ isEmpty: true });
+    acceleratorProfile.visit();
+    acceleratorProfile.findEmptyText().should('exist');
+    acceleratorProfile.findAddButton().should('be.enabled');
+  });
+
+  it('list accelerator profiles', () => {
+    initIntercepts({});
+    acceleratorProfile.visit();
+    acceleratorProfile.getRow('Name', 'TensorRT');
+    acceleratorProfile
+      .getRow('Name', 'TensorRT')
+      .findDescription()
+      .contains('Lorem, ipsum dolor sit amet consectetur adipisicing elit. Saepe, quis');
+    acceleratorProfile.getRow('Identifier', 'nvidia.com/gpu');
+    acceleratorProfile.findTableHeaderButton('Identifier').click();
+    acceleratorProfile.sortedByColumn('Identifier', 'asc');
+    acceleratorProfile.findTableHeaderButton('Identifier').click();
+    acceleratorProfile.sortedByColumn('Identifier', 'desc');
+    acceleratorProfile.findCreateButton();
```

Comment:
Should at least assert some state of the create button.
```suggestion
    acceleratorProfile.findCreateButton().should('be.enabled');
```